### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by victortoso**
**What this PR does / why we need it**:
I've found those by reading the code while working on #4089 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by victortoso** [comment]
Perhaps I should merge #3523 here?

File: pkg/virt-launcher/virtwrap/api/converter.go [hunk 1]
```diff
@@ -49,13 +49,11 @@ import (
 )
 
 const (
-	CPUModeHostPassthrough = "host-passthrough"
-	CPUModeHostModel       = "host-model"
```
Comment (vladikr) [inline pkg/virt-launcher/virtwrap/api/converter.go:53]: Aren't these needed?

**Comment by vladikr** [comment]
some tests are failing since `monitorLoop` is missing.

```
15:48:34: /root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/2171/execroot/__main__/pkg/virt-launcher/monitor_test.go:176:9: mon.monitorLoop undefined (type *monitor has no field or method monitorLoop)
15:48:34: /root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/2171/execroot/__main__/pkg/virt-launcher/monitor_test.go:204:9: mon.monitorLoop undefined (type *monitor has no field or method monitorLoop)
15:48:34: /root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/2171/execroot/__main__/pkg/virt-launcher/monitor_test.go:222:9: mon.monitorLoop undefined (type *monitor has no field or method monitorLoop)
```

**Comment by victortoso** [comment]
`make test` works here

> Executed 80 out of 80 tests: 80 tests pass.
> INFO: Build completed successfully, 875 total actions

Removed two commits as they were fixed by 303e09654  and 2c234e3e3

**Comment by victortoso** [comment]
> 04:04:37: 04:04:37: failed to fetch vmis: the server could not find the requested resource (get virtualmachineinstances.kubevirt.io)

Not sure what went wrong here.

**Comment by victortoso** [comment]
Rebased due conflict with 49fbce35a2b5f207ef

**Comment by victortoso** [comment]
Rebased, no conflicts.
Friendly ping.

**Pull request closed without merging**